src/Internals/WCCom/WCConnectionHelper.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Helpers for managing connection to WooCommerce.com.
+ *
+ * @package Automattic\WooCommerce\Internals\WCCom
+ */
+
+namespace Automattic\WooCommerce\Internals\WCCom;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class WCConnectionHelper.
+ *
+ * Helpers for managing connection to WooCommerce.com.
+ */
+final class WCConnectionHelper {
+	/**
+	 * Check if WooCommerce.com account is connected.
+	 *
+	 * @since 4.4.0
+	 * @return bool Whether account is connected.
+	 */
+	public static function is_wccom_connected() {
```

Comment:
How would you feel about just `is_connected` since the method is already in the WCCom connection helper?

Reply:
Renamed in https://github.com/woocommerce/woocommerce/pull/26932/commits/4500e86198729ce893945a39fe9e22cee3991856